addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -51,66 +56,67 @@ options.registry.mailing_list_subscribe = mass_mailing_common.extend({
 options.registry.newsletter_popup = mass_mailing_common.extend({
     popup_template_id: "editor_new_mailing_list_subscribe_popup",
     popup_title: _t("Add a Newsletter Subscribe Popup"),
-    select_mailing_list: function (previewMode, value) {
+    /**
+     * @override
+     */
+    start: function () {
         var self = this;
-        return this._super(previewMode, value).then(function (mailing_list_id) {
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'read',
-                args: [[parseInt(mailing_list_id)], ['popup_content'], self.options.recordInfo.context],
-            }).then(function (data) {
-                self.$target.find(".o_popup_content_dev").empty();
-                if (data && data[0].popup_content) {
-                    $(data[0].popup_content).appendTo(self.$target.find(".o_popup_content_dev"));
-                }
-            });
+        this.$target.on('click', '.o_edit_popup', function (ev) {
+            ev.stopPropagation();
```

Comment:
Comment still applis ;)

Reply:
In our case, `deactivate_snippet` is called before the snippet is activated  from here: [Active snippet editor on click](https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/wysiwyg_snippets/snippets.editor.js#L635)
due to this, we have used ` ev.stopPropagation();` and if don't use this it will generate issue like this see image given image for more.
![newsletter](https://user-images.githubusercontent.com/22187704/59348509-f2a18c00-8d34-11e9-8ca5-1f2101bbe0bb.gif)

Reply:
Add a comment next to the line: `// So that the snippet is not enabled again by the editor`